docs/baseplate/context/index.rst

```diff
@@ -20,7 +20,7 @@ DIY: The Factory
 ----------------
 
 If a library you want isn't supported here, it can be added to your own
-application by subclassing :py:class:`~baseplate.context.ContextFactory`.
+application by implementing :py:class:`~baseplate.context.ContextFactory`.
```

Comment:
Implementing as in ContextFactory is some kind of interface? sub-classing/inherting made sense to me if we're just talking about inheriting and using a utility class, unless we are expected to override an implement a specific set of methods.